save-demo/src/main/kotlin/com/saveourtool/save/demo/service/GithubDownloadToolService.kt

```diff
@@ -109,19 +118,29 @@ class GithubDownloadToolService(
                 }
         }
         .flatMap { key ->
-            downloadFromGithubAndUploadToStorage(key).toMono()
+            downloadFromGithubAndUploadToStorage(
+                GithubRepo(key.ownerName, key.toolName),
+                key.vcsTagName,
+            ).toMono()
```

Comment:
suggestion to use `toMono` only with constants (pre-calculated values)

Reply:
Oh, sure

Reply:
Done